crates/polars-plan/src/dsl/string.rs

```diff
@@ -455,9 +455,17 @@ impl StringNameSpace {
     }
 
     /// Remove leading and trailing characters, or whitespace if matches is None.
-    pub fn strip(self, matches: Option<String>) -> Expr {
+    pub fn strip_chars(self, matches: Option<String>) -> Expr {
         self.0
-            .map_private(FunctionExpr::StringExpr(StringFunction::Strip(matches)))
+            .map_private(FunctionExpr::StringExpr(StringFunction::StripChars(
+                matches,
+            )))
+    }
+
+    /// Remove leading and trailing characters, or whitespace if matches is None.
+    #[deprecated(since = "0.32.1", note = "use polars strip_chars")]
```

Comment:
Not entirely sure what should go in the `note` here, please correct me if this is wrong